src/BTVNanoCommissioning/workflows/ctag_Wctt_valid_sf.py

```diff
@@ -626,6 +647,17 @@ def process_shift(self, events, shift_name):
                     ak.values_astype(events[event_level].Pileup.nTrueInt, int),
                     weight=weight,
                 )
+            if "Wc" in self.selMod:
```

Comment:
Could you please add a comment this is for WP-based SF? Also, from the `if` statement here, it seems to me you want to add the WP histogram for all the W+c workflow including shape based, did I understand correctly?

Reply:
For fixed WP-based SFs these histograms are used for the analysis. We could also remove from shape analysis.

Reply:
I think either way is fine :p, just not used in the shape based. I want to cross check